README.md

```diff
@@ -81,7 +81,7 @@ We provide multiple deployment methods, but it is recommended to use our Ansible
 
 ### Testing deployment
 
-- [Try TiKV and TiDB](https://github.com/pingcap/docs/blob/master/dev/how-to/get-started/deploy-tidb-from-docker-compose.md)
+- [Try TiKV and TiDB](./docs/how-to/get-started/deploy-using-docker-compose.md)
```

Comment:
```suggestion
- [Try TiKV and TiDB](https://tikv.org/docs/3.0/tasks/try/)
```

Reply:
We are going to remove the docs folder soon because we don't maintain it.

Reply:
OK updating it..